cli/src/cli_util.rs

```diff
@@ -345,6 +345,20 @@ impl CommandHelper {
         &self.data.settings
     }
 
+    /// Resolves configuration for new workspace located at the specified path.
```

Comment:
Maybe add comment explaining that it may be different due to conditional config?

Reply:
Updated the doc to mention conditional variables and repo config file.